doc/api/crypto.md

```diff
@@ -1548,7 +1548,7 @@ If `privateKey` is not a [`KeyObject`][], this function behaves as if
 object, the following additional properties can be passed:
 
 * `dsaEncoding` {string} For DSA and ECDSA, this option specifies the
-  format of the generated signature. It can be one of the following:
+  format of the signature. It can be one of the following:
```

Comment:
This usage is correct and the change should be reverted. `sign()` does actually generate a signature.

```suggestion
  format of the generated signature. It can be one of the following:
```